docs/reference/languagedetails.md

```diff
@@ -78,7 +78,15 @@ future.
 
 ### Characters
 
-In the future, Dark will support individual characters.
+Characters in Dark are single display characters. Technically, they are Extended
+Grapheme Clusters, and support `'a'`, `'1'`, `'Ż'`, `'🇵🇷'`, `'👩‍👩‍👧‍👦'`, etc. See
+[Strings](###Strings) above for additional context.
+
+While the Dark language has sufficient support for Characters, the Editor
```

Comment:
I feel we're telling people to use `'` and then telling them they can't. I'd approach this as:

"Dark currently supports:
- x
- y
- z

Dark does not yet support:
- a
- b
- c
"